### DISCUSSION TITLE
Link is broken from TEALS GitHub main page

### DISCUSSION THREAD

**Discussion by jason44107** [TEALSK12/apcsa-public#290] [General]
The [TEALS GitHub main page](https://tealsk12.github.io/forums.html) has a link to "AP Computer Science A," but that link doesn't work because there's a typo in the URL.  The link's URL starts with "hhttps" but it should be "https" -- remove the extra h.  I've opened an issue for this.  Thank you.

**Comment by menglish87** [comment]
Hi Jason,

Thanks so much for flagging this and for opening an issue! This should be resolved shortly if it hasn't been resolved already.

Best,
Michael English
